docs/cli.md

```diff
@@ -8,25 +8,29 @@ last_update:
 license: 'This software is licensed under the Apache License version 2.'
 ---
 
-This document provides guidance on managing metadata within Apache Gravitino using the Command Line Interface (CLI). The CLI offers a terminal based alternative to using code or the REST interface for metadata management.
+This document provides a guidance on managing metadata within Apache Gravitino using
+the Command Line Interface (CLI). The CLI offers a terminal based alternative to
+using code or the REST API for metadata management.
 
-Currently, the CLI allows users to view metadata information for metalakes, catalogs, schemas, tables, users, roles, groups, tags, topics and filesets. Future updates will expand on these capabilities.
+Currently, the CLI allows users to view metadata information for metalakes, catalogs,
+schemas, tables, users, roles, groups, tags, topics and filesets.
+Future updates will expand on these capabilities.
 
 ## Running the CLI
 
-You can configure an alias for the CLI for ease of use, with the following command:
+You can configure an alias for the CLI for ease of use. For example:
```

Comment:
This is not an example.

Reply:
Well .. it is. It is using a "incubating", "SNAPSHOT" jar.
If the syntax is changed to `alias gcli='java -jar <path-to-jar>`, I would not call it an example.

Reply:
Because if you compile it, that is what you always get, but it's a minor thing. This change probably doesn't matter in teh scheme of things.

Reply:
Okay. I'm not strongly opinionated at this.

Reply:
Neither am I